src/status_im/chat/events.cljs

```diff
@@ -347,3 +343,19 @@
         (assoc :delete-chat chat-id)
         (not debug?)
         (assoc :deactivate-chat chat-id)))))
+
+(handlers/register-handler-fx
+  :delete-chat
+  [re-frame/trim-v]
+  (fn [cofx [chat-id]]
+    (-> (models/remove-chat cofx chat-id)
+        (update :db navigation/replace-view :home))))
+
+(handlers/register-handler-fx
+  :delete-chat?
```

Comment:
maybe `:show-delete-chat-confirmation`

Reply:
We don't use `?` yet but could be a convention for handler name? `-confirmation` looks pretty long :)